core/arch/arm/plat-synquacer/rng_pta.c

```diff
@@ -0,0 +1,211 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (C) 2018, Linaro Limited
+ */
+
+#include <io.h>
+#include <kernel/delay.h>
+#include <kernel/pseudo_ta.h>
+#include <mm/core_memprot.h>
+#include <string.h>
+#include <rng_pta.h>
+
+#define PTA_NAME "rng.pta"
+
+#define ENTROPY_POOL_SIZE	4096
+
+static uint8_t entropy_pool[ENTROPY_POOL_SIZE] = {0};
+static uint32_t entropy_size;
+
+static const uint32_t table[] = {
```

Comment:
Suggestion: Either rename this to `crc32_table` or add a comment above stating that this is a CRC32 table (a link to some spec would be nice).

Reply:
Will rename it to `crc32_table` with a link to some spec.